dashboard/test/ui/test_runner/test_runner_options_parser.rb

```diff
@@ -0,0 +1,210 @@
+require 'ostruct'
+require 'optparse'
+
+class TestRunnerOptionsParser
```

Comment:
I like that this file is in a `test_runner` directory, but I'd love it if that directory could also represent a module, and if we avoid repeating `test_runner` in the filenames. I'm thinking that we move this class to a file `dashboard/test/ui/test_runner/options_parser.rb` and structure it like so:

```suggestion
module TestRunner
  class OptionsParser
```

I think this would better mirror the kind of organization I see in codebases like Rails. What do you think?